pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -305,6 +308,12 @@ func managePod_v311_00_to_latest(ctx context.Context, configMapsGetter corev1cli
 	allFeatureGates := getFeatureGateString(sortedFeatureGates, featureGates)
 	required.Spec.Containers[0].Args = append(required.Spec.Containers[0].Args, fmt.Sprintf("--feature-gates=%v", allFeatureGates))
 
+	if len(os.Getenv("SCHEDULER_IMAGE")) != 0 {
```

Comment:
What is this for?

Reply:
Oh, I should remove this. I needed this because there is a problem with simply trying to set `${IMAGE}` for a custom operand image when you build a kube-scheduler that isn't hyperkube. Because we use `${IMAGE}` in the [initcontainer](https://github.com/openshift/cluster-kube-scheduler-operator/blob/master/bindata/v4.1.0/kube-scheduler/pod.yaml#L15) as well as the operand container our current logic breaks if you supply it a kube-scheduler image that doesn't have `wait-for-host-port`. Or it's something like that

Reply:
:stuck_out_tongue_winking_eye: